src/python/zquantum/qubo/conversions.py

```diff
@@ -10,6 +10,12 @@
 def convert_qubo_to_openfermion_ising(qubo: BinaryQuadraticModel) -> IsingOperator:
     """Converts dimod BinaryQuadraticModel to OpenFermion IsingOperator object.
 
+    The resulting Openfermion IsingOperator has the following property:
+    For every bitstring, its expected value is the same as the one of the original QUBO.
```

Comment:
I know I worded this, but now I think it should read: "For every bitstring, its expected value is the same as the energy of the original QUBO." WDYT @mstechly?